configs/swarm/docker-compose.local.yml

```diff
@@ -22,7 +22,12 @@ services:
     init: true
     command: |-
       /bin/sh -c "
-      kagami genesis sign /config/genesis.json --public-key $$GENESIS_PUBLIC_KEY --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 --out-file $$GENESIS_SIGNED_FILE &&
+      kagami genesis sign /config/genesis.json \\
+          --public-key $$GENESIS_PUBLIC_KEY \\
+          --private-key 80264082B3BDE54AEBECA4146257DA0DE8D59D8E46D5FE34887DCD8072866792FCB3AD4164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4 \\
+          --topology '[{\"address\":\"irohad0:1337\",\"public_key\":\"ed012082528CCC8727333530C8F6F19F70C23882DEB1BF2BA3BE4A6654C7E8A91A7731\"},{\"address\":\"irohad1:1338\",\"public_key\":\"ed012083C85E315776FD2DDC187ECB23E608F800B313A1D614B108078EC048D5013D2D\"},{\"address\":\"irohad2:1339\",\"public_key\":\"ed0120A37B7B758C952FE9429E9E35D1D71E2D8BB9364EDD077B5027ABAAC798D3230E\"},{\"address\":\"irohad3:1340\",\"public_key\":\"ed0120B23E14F659B91736AAB980B6ADDCE4B1DB8A138AB0267E049C082A744471714E\"}]' \\
```

Comment:
Does `--topology` is used to override what is already defined in `genesis.json`?

Reply:
I'm a bit concerned that `kagami genesis sign` is not single purpose tool.
Maybe we can use smt like `jq` to replace `"topology"` in separate step?

```
jq '.topology=[...]' /config/genesis.json > $$GENESIS_SIGNED_FILE
kagami genesis sign $$GENESIS_SIGNED_FILE ... --out-file $$GENESIS_SIGNED_FILE
```

Reply:
good point. This should be defined in `genesis.json`

Reply:
> Maybe we can use smt like `jq` to replace `"topology"` in separate step?

Good idea, but this requires to add `jq` to docker image, is it ok?
Alternatively I can add command like `kagami genesis replace-topology`

Reply:
Imo it’s okay, we might ask our DevOps

Reply:
Used jq to update topology and executor path